expense_tracker/lib/expenses_screen/expenses_list/exoense_item.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:expense_tracker/models/expense.dart';
+import 'package:flutter/material.dart';
+
```

Comment:
Just gonna say that this file has a small typo, it is called ex**o**ense_item but you probable meant ex**p**ense_item